### PR TITLE
Posting interface

### PR DESCRIPTION
**Pull request by jsurya**
image

Complete:
- Ability to post in Plaintext/Markdown
- Ability to post with any Visibility
- Default Visibility set to Public
- Styling

Todo:
- Recipient author functionality

### CODE REVIEW

File: login/templates/registration/register.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
 {% block title %}User Registration{% endblock %}
 {% block head %}User Registration{% endblock %}
 {% block content %}
-    <form method="post" action=".">{% csrf_token %}
-        <table border="0">
+     <form class="form-horizontal" method="post" action=".">{% csrf_token %}
```
Comment (klindenbach) [inline login/templates/registration/register.html:9]: Does this make the register table look less gross?

Comment (jsurya) [inline login/templates/registration/register.html:9]: Yes LOL

File: posts/urls.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+admin.autodiscover()
+
+from posts import views
+
+urlpatterns = patterns('',
+    # Examples:
+    # url(r'^$', 'bookmarks.views.home', name='home'),
+    # url(r'^blog/', include('blog.urls')),
+
+    url(r'^admin/', include(admin.site.urls)),
```
Comment (klindenbach) [inline posts/urls.py]: Why do you have a `/post/admin` endpoint?

Comment (jsurya) [inline posts/urls.py]: Sorry, residual code from elsewhere..

**Pull request merged**